### DISCUSSION TITLE
unit in front of data label on chart header?

### DISCUSSION THREAD

**Discussion by yanggithub999** [RomRider/apexcharts-card#217] [Q&A]
hi there, 1st thanks for the hard work, this is one of the best chart for HA for sure.

A quick one, I use transform to get daily energy cost chart, but struggle to find a way for the chart header showing £2.1 instead 2.1£, any solution or it is not possible?

**Comment by yanggithub999** [comment]
<img width="510" alt="image" src="https://user-images.githubusercontent.com/90552782/135479603-c7faf9c0-021a-4be4-befb-0e0355721ed5.png">
this screenshot more clear to see where my question related.

thanks

**Comment by RomRider** [comment]
Please open a feature request for that.

**Comment by nobodyspecial** [comment]
Was this ever added?

**Reply by kay1010100** [reply]
> Was this ever added?

there is a unit variable
```

series:
  - entity: sensor.nordpool
    type: column
    unit: €/kWh
    float_precision: 3

```

![image](https://github.com/user-attachments/assets/9d3139bd-34ed-484b-8ff9-33aaaa705d68)

**Reply by calvinbui** [reply]
i dont think so, can't find anything about it either

**Comment by jbuggie** [comment]
would love to see a solution to this problem

**Comment by vingerha** [comment]
I am trying to pick up on things in my own repo...but before I spent (free) time, I'd like to know what are the use case(s?) as in all my work, the unit always follows the figure (fafaik)

**Comment by vingerha** [comment]
Thought it was simple, then made some weird errors.. anyhow
you can try this https://github.com/vingerha/apexcharts-card/releases/tag/v1.3.0
unit is boolean or 'before_value' and default: true

<img width="952" height="518" alt="image" src="https://github.com/user-attachments/assets/342f58f3-6e7b-42fa-a7d9-66cd4aa6eee7" />

**Reply by jbuggie** [reply]
In my use case, I need to display the $ sign before the numbers (as in money, $2.50)

**Reply by nobodyspecial** [reply]
> In my use case, I need to display the $ sign before the numbers (as in money, $2.50)

same here

**Comment by vingerha** [comment]
@jbuggie and @nobodyspecial ...yeah, forgot about those ...anyhow.. it works in my repo (above)...this one seems dead
